airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/PostgresDestination.java

```diff
@@ -73,7 +73,7 @@ protected DataSourceFactory.DataSourceBuilder modifyDataSourceBuilder(final Data
     // Adding 60 seconds to connection timeout, for ssl connections, default 10 seconds is not enough
     return builder.withConnectionTimeout(Duration.ofSeconds(60))
         .withConnectionInitSql("""
-                               CREATE FUNCTION pg_temp.airbyte_safe_cast(_in text, INOUT _out ANYELEMENT)
+                               CREATE OR REPLACE FUNCTION pg_temp.airbyte_safe_cast(_in text, INOUT _out ANYELEMENT)
```

Comment:
You need to bump the metadata and doc chagelog version too.

Reply:
You mean I should update this file:
https://github.com/airbytehq/airbyte/blob/master/.bumpversion.cfg
as a patch (last number) based on (https://semver.org/)

Or this file?
https://github.com/airbytehq/airbyte/blob/master/docs/integrations/destinations/postgres.md

Reply:
Gonna follow the files modified in this file and manually update them
https://github.com/airbytehq/airbyte/pull/37660/files